### ISSUE TITLE
UmbracoRichTextParser  - not respecting relative paths in MediaDomain

### ISSUE THREAD

**Issue by Malthebp** [enterspeedhq/enterspeed-source-umbraco-cms#123]
Hey Enterspeed.

The UmbracoRichTextParser does not respect relative paths in the media domain set in the Umbraco backoffice. 

It should be as simple as joining the 2 URL paths.

Something like:
```
...
foreach (var imageNode in imageNodes)
{
    var src = imageNode.GetAttributeValue("src", string.Empty);
    if (src.StartsWith("/media/"))
    {
        var relativePath = Path.Join(mediaDomainUrl.AbsolutePath, src);

        imageNode.SetAttributeValue("src", new Uri(mediaDomainUrl, relativePath).ToString());
    }
}
...
```

Regards
Malthe

**Comment by emilras** [comment]
Thank you @Malthebp! We will look into it :)

**Comment by jesperweber** [comment]
Hi @Malthebp, the fix is now release in version 3.0.1 of the package. 

Once again thank you for reporting this and given the solution as well :-)
